influxdb/files/influxdb.conf

```diff
@@ -187,7 +187,11 @@ reporting-disabled = false
 [http]
   enabled = true
   bind-address = "{{ server.http.bind.address }}:{{ server.http.bind.port }}"
+  {%- if server.admin_user is defined %}
```

Comment:
Ok. I will add the user into the admin section but lets add support for multiple admin in another PR.